### ISSUE TITLE
Point instead of Bounding Boxes

### ISSUE THREAD

**Issue by LucasPilla** [ultralytics/yolov5#8163]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.

### Description

Is there any version of YOLO for detecting a point instead of bounding boxes? Some applications does not require the complete localization and providing bounding boxes for each object is time consuming, just a keypoint would be enought.  [Here](https://openaccess.thecvf.com/content_CVPR_2019/papers/Ribera_Locating_Objects_Without_Bounding_Boxes_CVPR_2019_paper.pdf) a better explanation about this idea and some examples.

### Use case

With this feature labelling the images would be faster and having just the center point as output, for exemple, would be enought for some applications as weed detection and selective spraying.

### Are you willing to submit a PR?

- [X] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
@LucasPilla yes that's a good question. This is certainly possible with a bit of customization: you'd need different style labels, dataloaders and loss function, as well as visualizations. The model architecture would be mostly identical however, you'd just remove 2 values from each output, the width and height.
